services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -25,12 +25,28 @@ export const ReportPageWrapper = () => {
   } = useStore();
   const { reportType, state, reportId, pageId } = useParams();
   const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [hideElements, setHideElements] = useState<boolean>(false);
   const methods = useForm({
     defaultValues: {},
     shouldUnregister: true,
     resolver: yupResolver(elementsValidateSchema),
   });
 
+  useEffect(() => {
+    const formValues = methods.getValues() as any;
+    if (Object.keys(formValues).length === 0) {
+      return;
+    }
+    const reportingRadio = formValues?.elements?.find((element: any) => {
+      return element?.id === "measure-reporting-radio";
```

Comment:
I still am very hesitant about this component caring about specific ids. Is there instead a way to make the element care about the id it is controlled by instead?

This report wrapper will be used for future reports and adding special cases to it will confuse future development.